documentation/asciidoc/computers/getting-started/network-installation.adoc

```diff
@@ -51,11 +51,11 @@ You will need a keyboard to make use of the network installation feature. While
 
 video::b1SYVpM9lto[youtube]
 
-You should also insert a **blank** SD Card into the Raspberry Pi's SD Card slot.
+You should also insert a **blank** SD Card into the Raspberry Pi computer SD Card slot.
```

Comment:
"the Raspberry Pi computer SD Card slot." is not good grammar, maybe "the SD card slot of your Raspberry Pi."